<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoNav</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="targetSourceWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="targetSourceMenuLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Target Menu</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="2">
          <widget class="QToolButton" name="toolButton_11">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/images/address.bmp</normaloff>:/images/images/address.bmp</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="arrowType">
            <enum>Qt::RightArrow</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="addressButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Address</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/images/address.bmp</normaloff>:/images/images/address.bmp</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="mapButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Via Map</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/images/choosetarget.bmp</normaloff>:/images/images/choosetarget.bmp</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="bookmarksButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Bookmarks</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/images/bookmark.bmp</normaloff>:/images/images/bookmark.bmp</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="gpsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>GPS</string>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="arrowType">
            <enum>Qt::UpArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="mainMenuWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Main Menu</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QToolButton" name="browseButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Browse Map</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/images/viewmap.bmp</normaloff>:/images/images/viewmap.bmp</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="routeButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>View Route</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/images/back.bmp</normaloff>:/images/images/back.bmp</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="settingsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Settings</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/images/settings.bmp</normaloff>:/images/images/settings.bmp</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QToolButton" name="sourceMenuButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Source</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/images/source.bmp</normaloff>:/images/images/source.bmp</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="targetMenuButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Target</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/images/images/choosetarget.bmp</normaloff>:/images/images/choosetarget.bmp</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="settingsWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolButton" name="systemSettingsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>System</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="rendererSettingsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Map Renderer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="gpsLookupSettingsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>GPS Lookup</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="addressLookupSettingsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Address Lookup</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="gpsSettingsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>GPS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="dataDirectoryButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Data Directory</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="arrowType">
            <enum>Qt::LeftArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>targetMenuButton</sender>
   <signal>clicked()</signal>
   <receiver>targetSourceWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sourceMenuButton</sender>
   <signal>clicked()</signal>
   <receiver>targetSourceWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>targetMenuButton</sender>
   <signal>clicked()</signal>
   <receiver>mainMenuWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sourceMenuButton</sender>
   <signal>clicked()</signal>
   <receiver>mainMenuWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_11</sender>
   <signal>clicked()</signal>
   <receiver>targetSourceWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_11</sender>
   <signal>clicked()</signal>
   <receiver>mainMenuWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settingsButton</sender>
   <signal>clicked()</signal>
   <receiver>settingsWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settingsButton</sender>
   <signal>clicked()</signal>
   <receiver>mainMenuWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>settingsWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>mainMenuWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
